### PR TITLE
Usage Documentation Updates - cnoon Edits

### PR DESCRIPTION
**Pull request by cnoon**
This PR makes a few updates and edits to the Usage documentation PR.

### CODE REVIEW

File: Documentation/Usage.md [hunk 1]
```diff
@@ -197,20 +199,22 @@ let parameters: [String: [String]] = [
...
 AF.request("https://httpbin.org/post", method: .post, parameters: parameters, encoder: URLEncodedFormParameterEncoder.default)
 AF.request("https://httpbin.org/post", method: .post, parameters: parameters, encoder: URLEncodedFormParameterEncoder(destination: .httpBody))
+
 // HTTP body: "qux[]=x&qux[]=y&qux[]=z&baz[]=a&baz[]=b&foo[]=bar"
 ```
 
 ##### Configuring the Encoding of `Array` Parameters
 
-Since there is no published specification for how to encode collection types, by default Alamofire follows the convention of appending `[]` to the key for array values (`foo[]=1&foo[]=2`), and appending the key surrounded by square brackets for nested dictionary values (`foo[bar]=baz`).
+Since there is no published specification for how to encode collection types, by default, Alamofire follows the convention of appending `[]` to the key for array values (`foo[]=1&foo[]=2`), and appending the key surrounded by square brackets for nested dictionary values (`foo[bar]=baz`).
```
Comment (jshier) [inline Documentation/Usage.md]: This comma doesn't seem necessary. Adding one seems to make the clause part of the preceding statement.

File: Documentation/Usage.md [hunk 2]
```diff
@@ -482,7 +487,7 @@ AF.request("https://httpbin.org/get").response { response in
 
 #### Response Data Handler
 
-The `responseData` handler uses a `DataResponseSerializer`  to extract and validate the `Data` returned by the server. If no errors occur and `Data` is returned, the response `Result` will be a `.success` and the `value` will be the `Data` returned from the server.
+The `responseData` handler uses a `DataResponseSerializer`to extract and validate the `Data` returned by the server. If no errors occur and `Data` is returned, the response `Result` will be a `.success` and the `value` will be the `Data` returned from the server.
```
Comment (jshier) [inline Documentation/Usage.md]: Now there's no space between `DataResponseSerializer` and `to`.

File: Documentation/Usage.md [hunk 3]
```diff
@@ -55,16 +55,17 @@ Alamofire provides an elegant and composable interface to HTTP network requests.
 Additionally, networking in Alamofire (and the URL Loading System in general) is done _asynchronously_. Asynchronous programming may be a source of frustration to programmers unfamiliar with the concept, but there are [very good reasons](https://developer.apple.com/library/ios/qa/qa1693/_index.html) for doing it this way.
 
 #### Aside: The `AF` Namespace
-Previous versions of Alamofire's documentation used examples like `Alamofire.request()`. This API, while it appeared to require the `Alamofire` prefix, in fact worked fine without it. The `request` method and other functions were available globally in any file with `import Alamofire`. Starting in Alamofire 5, this functionality has been moved out of the global [namespace](https://en.wikipedia.org/wiki/Namespace) and into the `AF` enum, which acts as a namespace. This allows Alamofire to offer the same convenience functionality while not having to pollute the global namespace every time Alamofire is used. Similarly, types extended by Alamofire will use an `.af.` extension prefix to separate the functionality Alamofire offers from other extensions.
```
Comment (cnoon) [inline Documentation/Usage.md]: Just tweaked the last sentence here around the property extension.

File: Documentation/Usage.md [hunk 4]
```diff
@@ -55,16 +55,17 @@ Alamofire provides an elegant and composable interface to HTTP network requests.
...
+Previous versions of Alamofire's documentation used examples like `Alamofire.request()`. This API, while it appeared to require the `Alamofire` prefix, in fact worked fine without it. The `request` method and other functions were available globally in any file with `import Alamofire`. Starting in Alamofire 5, this functionality has been moved out of the global [namespace](https://en.wikipedia.org/wiki/Namespace) and into the `AF` enum, which acts as a namespace. This allows Alamofire to offer the same convenience functionality while not having to pollute the global namespace every time Alamofire is used. Similarly, types extended by Alamofire will use an `af` property extension to separate the functionality Alamofire adds from other extensions.
 
 ## Making Requests
-Alamofire provides a variety of convenient methods for making HTTP requests. At the simplest, just provide a `String` that can be converted into a `URL`:
```
Comment (cnoon) [inline Documentation/Usage.md]: Could go either way here, thought `convenience` seemed better in this context.

File: Documentation/Usage.md [hunk 5]
```diff
@@ -482,7 +487,7 @@ AF.request("https://httpbin.org/get").response { response in
 
 #### Response Data Handler
 
-The `responseData` handler uses a `DataResponseSerializer`  to extract and validate the `Data` returned by the server. If no errors occur and `Data` is returned, the response `Result` will be a `.success` and the `value` will be the `Data` returned from the server.
```
Comment (cnoon) [inline Documentation/Usage.md]: Weird, looks like my change removed the space altogether. Will fix.

Comment (cnoon) [reply Documentation/Usage.md]: Fixed in 68676b8d.

File: Documentation/Usage.md (hunk 2 above)
Comment (cnoon) [reply Documentation/Usage.md]: Yep, that was my bad, fixed in 68676b8d.

File: Documentation/Usage.md (hunk 1 above)
Comment (cnoon) [reply Documentation/Usage.md]: Removed in c5ec5f7c.

File: Documentation/Usage.md (hunk 4 above)
Comment (jshier) [reply Documentation/Usage.md]: Looks good.

File: Documentation/Usage.md (hunk 3 above)
Comment (jshier) [reply Documentation/Usage.md]: Makes sense.

**Pull request merged**